pages/resources/couseling.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function couseling() {
```

Comment:
Probably will update when complete, but make sure the component names are capitalized

Reply:
Updated!